src/pages/passwordResetPage.js

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import { resetPasword } from '@modules/resetpassword/actions';
+import SuccessContainer from '@components/common/Success';
+import Layout from '@components/common/Layout';
+import SyncLoader from 'react-spinners/SyncLoader';
+import { Formik, Form, Field } from 'formik';
+import * as Yup from 'yup';
+
+const passwordSchema = Yup.object().shape({
+  password: Yup.string()
```

Comment:
Consider using something like this instead:

```
password: Yup.string()
    .trim()
    .min(8, 'minumum of 8 characters')
    .matches(
      /^(?=.{8,}$)(?=.*[a-z])(?=.*[A-Z])(?=.*[0-9])(?=.*\W).*$/,
      'Password must contain atleast one special character and uppercase letter'
    )
    .required('Required'),
```
The password should not just be a minimum of 8 characters but must have an uppercase letter and a special character in it. This would allow it be in sync with password used during signup